gitpod-io/gitpod
# [preview-env] fix certification error and dns issue

## Description


![image](https://user-images.githubusercontent.com/20944364/196623973-4d5864f0-9af4-441d-8d90-b569106fe191.png)

In https://github.com/gitpod-io/gitpod/pull/13953 PR, we switch to use `dev` as our preview environment shortname, but we forgot to adjust dns and cert. This PR fixes that.

After merged, old PR need run with `with-clean-slate-deployment=true`, otherwise it will not execute terraform

## Related Issue(s)

Fixes #

## How to test

1. start a workspace in this preview environment https://hw-fix-dev-cert-err.preview.gitpod-dev.com/workspaces
2. ensure the url is `workspaceid.ws-dev.*`
3. make sure there is no cert error
4. open vscode desktop to check ssh gateway works

<img width="1690" alt="image" src="https://user-images.githubusercontent.com/20944364/196663333-c2f3b6d6-2c42-4961-a268-024ec6a5483a.png">

<img width="1028" alt="image" src="https://user-images.githubusercontent.com/20944364/196663693-eff67281-aba7-4f7a-9467-14cd1f053fc4.png">



## Release Notes

```release-note
NONE
```

## Documentation


## Werft options:

      If enabled this will build `install/preview`
      Valid options are `all`, `workspace`, `webapp`, `ide`
